.travis.yml

```diff
@@ -66,10 +68,14 @@ matrix:
            CONDA_DEPENDENCIES=""
            INSTALL_HOLE="false"
 
+    # pin dependencies for legacy:
+    # - ensure that chemfiles can get the last netcdf4 that is still supported
+    #   see https://github.com/MDAnalysis/mdanalysis/pull/2798#issuecomment-679991785
     - env: NAME="python 2.7"
            PYTHON_VERSION=2.7
            CODECOV="true"
            NUMPY_VERSION=1.16
+           CONDA_DEPENDENCIES="${CONDA_DEPENDENCIES} chemfiles-lib=0.9.3=he1b5a44_0"
```

Comment:
Is this the correct way to pin chemfiles-lib?

Reply:
Yes, that should be. I only tested installing chemfiles + netcdf4 this way, so there might be additional interaction with other dependencies. I'll have another look =)

Reply:
Thank you, much appreciated!!

Reply:
I don't understand fully what's happening here (I can't get conda to reproduce the bug locally, so I have to start Travis to check, which is very slow 😃); but I got the tests to work with Python 2.7 on this branch by using the PyPi/pip version of chemfiles instead of the conda one (https://travis-ci.org/github/Luthaf/mdanalysis/jobs/722309790). Basically adding chemfiles to `PIP_DEPENDENCIES` and removing it from `CONDA_DEPENDENCIES`.

The main difference with conda is that I'm building the wheels on PyPi myself instead of using the conda tools, but the linux wheel is built with the [manylinux1 docker image](https://github.com/pypa/manylinux), which should guarantee compatibility with all possible Python installations on Linux. The wheels also work perfectly on osx & windows.